pypuf/learner/regression/logistic_regression.py

```diff
@@ -227,7 +226,4 @@ def learn(self):
             distance = (self.training_set.N - count_nonzero(self.training_set.responses == self.sign_combined_model_responses)) / self.training_set.N
             self.min_distance = min(distance, self.min_distance)
 
-        if not converged and distance > .01:
-            stderr.write('\rNOT CONVERGED                    ')
```

Comment:
I can see three options for this:

1. Use the logger from tools.py and probably mess up the multiprocessing log.
2. Set a flag as a member variable and check it when analysing the learner.
3. Create a nifty logging server which understands the current scopes of logging and handles requests correct.

What shall I do?